estate/views/estate_menus.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<odoo>
+
+  <menuitem id="estate_menu" action="estate_property_menu">
```

Comment:
```suggestion
  <menuitem id="estate_menu" name="Real Estate">
```
I think this action is useless, the "root" menu (a menu without parent) will always open the related App.
See comment below regarding the two actions.